cmd/exit.go

```diff
@@ -38,6 +38,7 @@ const (
 type Platform interface {
 	API() string
 	CodeFor(errType LifecycleExitError) int
+	SupportsAssetPackages() bool
```

Comment:
There is some interesting entanglement happening between cmd and platform packages - I am not sure what to do about it, maybe it's fine as it is. @jabrown85 @yaelharel any thoughts here?

Reply:
I don't love the entanglement happening. Why do we need `SupportsAssetPackages` in the cmd interface? Is it because we have end up sending `cmd.Platform` into a `lifecycle.Platform` function somewhere?

Reply:
It's [here](https://github.com/dwillist/lifecycle/blob/3ffc711f63cece48178021066f97b3ea9e636ac6/cmd/lifecycle/builder.go#L92) in cmd/lifecycle/builder.go: `env.NewBuildEnv(os.Environ(), ba.platform)`